docs/zkapps/o1js/interact-with-mina.md

```diff
@@ -199,75 +246,57 @@ What's interesting is that the transaction now has 3 account updates:
-
-In general, there are three kinds of authorizations that an account update can have: a proof, a signature, or none. We'll learn about signatures in the next section.
+- You didn't explicitly create the receiver account update. It was created, and attached to the transaction, by calling `this.send()`. 
 
-<!-- TODO: link to permission section when it exists -->
+  o1js tries to abstract away the low-level language of account updates where possible and give you intuitive commands to create the right ones. However, you might sometimes have to create account updates explicitly.
 
-Next, we observe that the user account update has a `callDepth: 1` and a non-default `caller` field. We won't explain this in detail, but it has to do with the fact that it was created from within a zkApp call. Account updates, even though displayed as a flat list here, are implicitly structured as a _list of trees_. Updates with a call depth of 1 or higher are child nodes of another update in that list of trees. In our case, the zkApp (sender) account update is at the top level (`callDepth: 0`) and the user (receiver) account update is a child of it.
+- The user update has `authorizationKind: 'None_given'`. That means the update is not authorized. This is possible because it doesn't include any changes that require authorization: It just receives MINA. You can send someone MINA without their permission.
 
-So, what is the meaning of this tree structure? Recall that in the last section, we explained how the zkApp account update is public input to its proof. Now, the fully general version of that statement is:
-**In a tree of account updates, all nodes are public inputs to the proof of the root node.** (If there is such a proof. This also holds for sub-trees of each tree.)
+In general, there are three kinds of authorizations that an account update can have: a proof, a signature, or none. We'll learn about signatures in the next section. You can find more details in [our docs about permissions](TODO).
 
-<!-- TODO: _ideal_ here would be a little picture of a tree, with a parent and a child node -->
 
-Concretely, in our example, both the zkApp account update and the user account update are public input to the zkApp method call. Intuitively, being public input means that the zkApp can "see" and constrain the update as part of its proof. Here, it means that nobody could change the public key of the receiver, or amount they receive, without making the proof invalid. The update can only contain what the method specified.
+- Proof authorization – used for zkApp accounts when you do an `@method` call. Proofs are verified against the on-chain verification key.
+- Signature authorization – used to update user accounts. Signatures are verified against the account's public key.
+- No authorization – used on updates which don't require authorization. For example, positive balance changes.
```

Comment:
As mentioned [here](https://github.com/o1-labs/docs2/pull/652#discussion_r1361706865) I think we should remove this
```suggestion
```